prime-router/metadata/fhir_mapping/hl7/datatypes/CX/Identifier.yml

```diff
@@ -0,0 +1,177 @@
+resourceType: Identifier
+
+value:
+  type: STRING
+  valueOf: CX.1
+  expressionType: HL7Spec
+
+extension_1:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: identifier-check-digit
+    valueString:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CX.2
+
+extension_2:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: naming-system-check-digit
+    valueString:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CX.3
+
+# Checking if HD is an Organization or a System by looking at the ID type.
+# If ID type is null or ID type is not ISO or UUID, treat as organization.
+# Otherwise, treat as system.
+assigner_1:
+  vars:
+    cx43: String, CX.4.3
+  condition: cx43 NULL
+  valueOf: datatypes/HD/Organization
+  expressionType: reference
+  specs: CX.4
+
+assigner_2:
+  vars:
+    cx43: String, CX.4.3
+  condition: $cx43 NOT_NULL && $cx43 NOT_EQUALS ISO && $cx43 NOT_EQUALS UUID
+  valueOf: datatypes/HD/Organization
+  expressionType: reference
+  specs: CX.4
+
+system:
+  vars:
+    cx41: String, CX.4.1
+    cx43: String, CX.4.3
+  expressionType: nested
+  condition: $cx43 EQUALS ISO || cx43 EQUALS UUID
+  expressions:
+    - condition: $cx41 NOT_NULL
+      expressionType: HL7Spec
+      type: STRING
+      valueOf: CX.4.1
+    - condition: $cx41 NULL && cx43 EQUALS ISO
+      expressionType: HL7Spec
+      type: STRING
+      constants:
+        prefix: "urn:oid:"
+      vars:
+        uri: String, CX.4.2
+        join: $prefix + $uri
+      valueOf: $join
+    - condition: $cx41 NULL && cx43 EQUALS UUID
+      expressionType: HL7Spec
+      type: STRING
+      constants:
+        prefix: "urn:uuid:"
+      vars:
+        uri: String, CX.4.2
+        join: $prefix + $uri
+      valueOf: $join
+
+extension_system:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: coding-system
+    valueCodeableConcept:
+      valueOf: datatype/CodeableConcept_var
+      expressionType: resource
+      vars:
+        code: String, CX.4
+
+type:
+  valueOf: datatype/CodeableConcept_var
+  expressionType: resource
+  vars:
+    code: $code
```

Comment:
I just realized there's kind of a conflict in the mapping here.. OBR.53 maps to an identifier with a `type.coding.code` of `PLAC`, but CX.5 also maps to the identifier's `type.coding.code`. In your test, CX.5 is `PLAC` too so it's fine, but if CX.5 is anything besides `PLAC` it won't be mapped to FHIR and it won't be in the output HL7. Not sure how we should handle that.. Did you already think this through? Might be good to talk through this case as a 17th at some point.

Reply:
I think it is a list of identifiers, so they should both be added.

Reply:
Yep, it's a list of identifiers, but within the OBR.53 identifier (the alternate-placer), I think there's only one `type.coding.code`, so if OBR.53.5 is anything other than `PLAC` it won't be preserved.

Reply:
Any updates on this topic? I would probably recommend adding a test that uses an OBR.53.5 of something other than `PLAC` for this to be more easily tracked.

Reply:
If I add a test with something other than PLAC it will fail because it will be overriden by PLAC. Kalish suggested putting CX.5 in an extension and getting it from there as a solution